### PR TITLE
[TECHNICAL SUPPORT] LPS-150923 Liferay.Service invokes Error Callback in case of empty response object

### PR DESCRIPTION
**Pull request by NemethNorbert**
Hey,
[LPS-150923](https://issues.liferay.com/browse/LPS-150923),

This issue is a regression of [LPS-142748](https://issues.liferay.com/browse/LPS-142748)
I removed the check for the length of the response object, as it does not tell us anything about wether its a good response or not, and we are already checking response.ok at line 202

Please review my changes,
Thanks

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js [hunk 1]
```diff
@@ -123,7 +123,6 @@ Liferay = window.Liferay || {};
 
 			ioConfig.complete = function (response) {
 				if (
-					Object.keys(response).length > 0 &&
```
Comment (georgel-pop-lr) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js:126]: Hi @NemethNorbert,

Have you checked if this condition was added for any reasons?

Regards.

Comment (georgel-pop-lr) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js:126]: > Hey @georgel-pop-lr ,
> 
> It was added in the linked ticket that caused the regression, trying to check wether the response is ok or not. While in the second commit, we decided to rely on response.ok as well, this change remained in the code. I tested [LPS-142748](https://issues.liferay.com/browse/LPS-142748) as well with my changes, the issue is not reproducible.

Okay, let's see what the team has to say, from my part this PR looks good.

Regards.

**Review by diegonvs — APPROVED** [review]
Nice! LGTM!

**Comment by diegonvs** [comment]
ci:forward

**Pull request closed without merging**